fdbserver/masterserver.actor.cpp

```diff
@@ -963,11 +963,22 @@ ACTOR Future<Void> readTransactionSystemState(Reference<MasterData> self,
 
 // Reads txnStateStore and broadcasts the data to all Commit Proxies. After
 // that, initializes Resolvers to allow recovery transaction goes through.
-ACTOR static Future<Void> sendInitialCommitToResolvers(Reference<MasterData> self) {
+ACTOR static Future<Void> sendInitialCommitToResolvers(Reference<MasterData> self, std::vector<std::pair<StorageServerInterface, ptxn::StorageTeamID>>* servers) {
 	state KeyRange txnKeys = allKeys;
 	state Sequence txnSequence = 0;
 	ASSERT(self->recoveryTransactionVersion);
 
+	for (const auto& pair : *servers) {
```

Comment:
These should be put inside block:
```
if (SERVER_KNOBS->TLOG_NEW_INTERFACE && self->lastEpochEnd == 0) {
}
```
i.e., only for the first time database starts and using PTXN tlogs.